### DISCUSSION TITLE
No video when external monitor is attached

### DISCUSSION THREAD

**Discussion by andynameistaken** [The-Wineskin-Project/WineskinServer#240] [Q&A] [answered]
Some games do not show any video when multiple screens are attached like:
- Gothic 1
- Star Trek: Voyager Elite Force

I am new to Wineskin if you want more information just write me what do you require.

**Comment by Gcenx** [comment, marked as answer]
This might simply be a problem with the games in question as I’ve ran two monitors and not seen this problem.
